packages/account/src/Components/financial-details/financial-details.tsx

```diff
@@ -9,38 +9,27 @@ import {
     Text,
     ThemedScrollbars,
 } from '@deriv/components';
-import { isDesktop, isMobile } from '@deriv/shared';
+import { isDesktop, isMobile, EMPLOYMENT_VALUES } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
-import { EMPLOYMENT_VALUES } from 'Constants/financial-details';
+import { TFinancialInformationForm } from 'Types';
 import FinancialInformation from './financial-details-partials';
 import { splitValidationResultTypes } from '../real-account-signup/helpers/utils';
 import ScrollToFieldWithError from '../forms/scroll-to-field-with-error';
 
-type TFinancialDetailsFormValues = {
-    income_source: string;
-    employment_industry: string;
-    occupation: string;
-    source_of_wealth: string;
-    education_level: string;
-    net_income: string;
-    estimated_worth: string;
-    account_turnover: string;
-};
-
 type TFinancialDetails = {
     goToPreviousStep: () => void;
     goToNextStep: () => void;
     getCurrentStep: () => number;
-    onSave: (current_step: number, values: TFinancialDetailsFormValues) => void;
+    onSave: (current_step: number, values: TFinancialInformationForm) => void;
     onSubmit: (
         current_step: number,
-        values: TFinancialDetailsFormValues,
+        values: TFinancialInformationForm,
         actions: (isSubmitting: boolean) => void,
         props: () => void
     ) => void;
     onCancel: (current_step: number, props: () => void) => void;
-    validate: (values: TFinancialDetailsFormValues) => object;
-    value: TFinancialDetailsFormValues;
+    validate: (values: TFinancialInformationForm) => object;
+    value: TFinancialInformationForm;
```

Comment:
can we please do one simple change renaming `value` to `initial_values` or `initial_form_values` to be more precise if these are the initial values for the form?

Reply:
Good idea, but this will be needed to be done from `packages/core/src/App/Containers/RealAccountSignup/account-wizard-form.js` for all the sections. Will consider it seperately